userspace/falco/yaml_helper.h

```diff
@@ -77,42 +78,77 @@ class yaml_helper
 		get_node(node, key);
 		if(node.IsDefined())
 		{
-			std::string value = node.as<std::string>();
+			auto value = node.as<std::string>();
 
 			// Helper function to convert string to the desired type T
 			auto convert_str_to_t = [&default_value](const std::string& str) -> T {
+				if (str.empty())
```

Comment:
Short-circuite, when empty value is passed, just return default.

Reply:
Do we have a test for it? I may have overlooked it.

Reply:
Will double check and eventually add one!